src/Cqm/QrdaControllers/QrdaReportController.php

```diff
@@ -228,28 +228,17 @@ public function downloadQrdaIII($pids, $measures = '', $options = []): void
 
         $directory = $GLOBALS['OE_SITE_DIR'] . DIRECTORY_SEPARATOR . 'documents' . DIRECTORY_SEPARATOR . 'cat3_reports';
         if (!is_dir($directory)) {
-            if (!mkdir($directory, 775, true) && !is_dir($directory)) {
+            if (!mkdir($directory, 0775, true) && !is_dir($directory)) {
                 throw new \RuntimeException(sprintf('Directory "%s" was not created', $directory));
             }
         }
 
-        if (is_array($pids)) {
-            if (count($pids) === 1) {
-                $pids = $pids[0];
-            } else {
-                $pids = '';
-            }
-        }
```

Comment:
this won't work. need for the request one.
I'll fix in my pr.

Reply:
k, sorry about that, you told me to remove it last night.  Haha, I shouldn't code late at night I guess.